web/src/repo/backend.ts

```diff
@@ -21,20 +21,24 @@ export const fetchRepository = memoizeObservable(
     (args: { repoName: string }): Observable<GQL.IRepository> =>
         queryGraphQL(
             gql`
-                query Repository($repoName: String!) {
-                    repository(name: $repoName) {
-                        id
-                        name
-                        url
-                        externalURLs {
+                query RepositoryRedirect($repoName: String!) {
+                    repositoryRedirect(name: $repoName) {
```

Comment:
You need to include this in the query if you want to use it below
```suggestion
                    repositoryRedirect(name: $repoName) {
                        __typename
```